pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -301,6 +309,12 @@ type mountedPod struct {
 	// fsResizeRequired indicates the underlying volume has been successfully
 	// mounted to this pod but its size has been expanded after that.
 	fsResizeRequired bool
+
+	// volumeMountStateForPod stores state of volume mount for the pod. if it is:
+	//   - VolumeMounted: means volume for pod has been successfully mounted
+	//   - VolumeMountUncertain: means volume for pod may not be mounted, but it must be unmounted
+	//   - VolumeNotMounted: means volume for pod has not been mounted
```

Comment:
do you need VolumeNotMounted state? I think the only action for this state is to delete the volume from the cache. This state is not used.

Reply:
I think we can drop it.